scripts/test/cod_documentation/style-guide.scss

```diff
@@ -751,14 +753,16 @@
 
 /**
 @Section
-  Image Layout
+  Image Layout &amp; Settings
```

Comment:
Is `&amp;` needed or would `&` work as well?

Reply:
I'm not..sure. Will `&` break it?

Reply:
@lwtchu Looks like you actually need to use `&`
![screenshot_41](https://user-images.githubusercontent.com/1285308/31235077-2110d8fc-a9b6-11e7-8988-b0ecc1bc6e01.jpg)

Reply:
Okay ..how do I undo this D: